### PR TITLE
Issue 195 memo text

### PR DESCRIPTION
**Pull request by cavazosjoe**

### CODE REVIEW

File: src/main/java/org/stellar/sdk/MemoId.java [hunk 1]
```diff
@@ -37,4 +37,9 @@ public boolean equals(Object o) {
     MemoId memoId = (MemoId) o;
     return id == memoId.id;
   }
+
+  @Override
+  public String toString() {
+    return Long.toString(id);
```
Comment (tamirms) [inline src/main/java/org/stellar/sdk/MemoId.java]: `id` is actually supposed to be interpreted as unsigned. Can you change this to:

```
import com.google.common.primitives.UnsignedLongs;
...
  @Override
  public String toString() {
    return UnsignedLongs.toString(this.id);
  }
```

Comment (cavazosjoe) [reply src/main/java/org/stellar/sdk/MemoId.java]: Sure, fixed

**Comment by tamirms** [comment]
@cavazosjoe it looks good! I apologize, I have already merged the release branch. Can you rebase your branch on top of master? Once you've done that I will merge

**Comment by cavazosjoe** [comment]
@tamirms merged master into my release branch, conflicts should be resolved now

**Pull request merged**